data/scenario-test.cfg

```diff
@@ -619,6 +619,13 @@ Adjacent own units of lower level will do more damage in battle. When a unit adj
         [/unit]
     [/side]
 
+    [label]
+        x,y=12,13
+        text="Jamming Orc"
+        tooltip="This unit’s ability reduces vision, that’s why 13,13 is shrouded at the start."
```

Comment:
I think I'd say "is initially shrouded" rather than "is shrouded at the start" but I guess it's not a big deal, both work.

Reply:
I like that and it does sound better for native speakers. However, I'm going to keep it as-is because I think keeping the language simpler is better for an obscure tooltip in a dev-only scenario.